### PR TITLE
fix: cleanup input on add liquidity

### LINKED ISSUE #1323: Value input in Add Liquidity does not clear itself from max value after going to other page
**Description**
Value input in Add Liquidity does not clear itself from max value after going to other page or opening another pair.

#### Definition of done
- [ ] Description is created (Task creator)
- [ ] Code is complete (Developer)
- [ ] Scope of test and what was done section is defined in task (Developer)
- [ ] Review passed (Code reviewer)
- [ ] Verified manually (QA)
- [ ] Test cases created/updated if necessary (QA)
- [ ] Change successfully merged
- [ ] Release notes added if needed

**Recording**

https://images.zenhubusercontent.com/209400598/344d4189-a0b8-4c29-ab8e-de5fa5e1d0ac/s.mp4

### PR DESCRIPTION
**Pull request by berteotti**
# Summary

Fixes #1323

 # To Test

1. Open the page `liquidity`
2. Input token amount
3. Go to another page
4. Reopen `liquidity` page
- [ ] Input should be cleaned

### CODE REVIEW

**Review by Diogomartf — APPROVED** [review]

File: src/pages/AddLiquidity/index.tsx [hunk 1]
```diff
@@ -304,6 +304,14 @@ export default function AddLiquidity() {
 
   const isCreate = location.pathname.includes('/create')
 
+  useEffect(() => {
+    //input cleanup on component unmount
```
Comment (Diogomartf) [inline src/pages/AddLiquidity/index.tsx:308]: Do we need the comment?

I would remove it. 
If we want to to be descriptive I would suggest make it a function and give it a descriptive name eg. 
`cleanUpInput`

**Comment by niemam29** [comment]
Tested and looks good to me

File: src/pages/AddLiquidity/index.tsx (hunk 1 above)
Comment (berteotti) [reply src/pages/AddLiquidity/index.tsx]: Updated to a named function

**Review by mix1o — APPROVED** [review]

**Review by jorgedclopes — APPROVED** [review]

**Comment by MilanVojnovic95** [comment]
Retested and looks ok to me 👍

**Pull request merged**